src/main/scala/org/broadinstitute/dsde/firecloud/model/Trial.scala

```diff
@@ -99,9 +101,68 @@ object Trial {
 
   }
 
-  case class TrialProject(name: RawlsBillingProjectName, verified: Boolean, user: Option[WorkbenchUserInfo])
+  case class TrialProject(name: RawlsBillingProjectName, verified: Boolean, user: Option[WorkbenchUserInfo], status: Option[CreationStatus])
   object TrialProject {
-    def apply(name: RawlsBillingProjectName) = new TrialProject(name, verified=false, user=None)
+    def apply(name: RawlsBillingProjectName) = new TrialProject(name, verified=false, user=None, status=None)
   }
 
+  case class CreateProjectsResponse(success: Boolean, count: Int, message: Option[String])
+
+  // following are horribly copied-and-pasted from rawls core, since they're not available as shared models
```

Comment:
Same comment - let's add a ticket for this.
Additionally, would it make sense to keep the rawls case classes together? Not a huge issue.